packages/core/lib/src/core_widget_factory.dart

```diff
@@ -188,6 +193,31 @@ class WidgetFactory {
     }
   }
 
+  /// Builds decoration image from [url]
+  DecorationImage? _buildDecorationImage(String? url) {
```

Comment:
This method should be public, so the svg package can override it to support SVG background.